nav2_behavior_tree/include/nav2_behavior_tree/compute_control_action.hpp

```diff
@@ -12,25 +12,25 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef NAV2_BEHAVIOR_TREE__FOLLOW_PATH_ACTION_HPP_
-#define NAV2_BEHAVIOR_TREE__FOLLOW_PATH_ACTION_HPP_
+#ifndef NAV2_BEHAVIOR_TREE__COMPUTE_CONTROL_ACTION_HPP_
+#define NAV2_BEHAVIOR_TREE__COMPUTE_CONTROL_ACTION_HPP_
 
 #include <memory>
 #include <string>
 
-#include "nav2_msgs/action/follow_path.hpp"
+#include "nav2_msgs/action/compute_control.hpp"
 #include "nav2_behavior_tree/bt_action_node.hpp"
 
 namespace nav2_behavior_tree
 {
 
-class FollowPathAction : public BtActionNode<nav2_msgs::action::FollowPath>
+class ComputeControlAction : public BtActionNode<nav2_msgs::action::ComputeControl>
```

Comment:
For example, given my previous feedback, this would be the `MoveBaseAction` BT action node.

Reply:
`MoveBaseAction`

This triggers me from ROS1. See other comment, this is all getting reverted to FollowPath since I have a feeling @mkhansen-intel will want it and its reasonable if we do one, we do both